packages/react-dataparcels/src/useParcelSideEffect.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import type ChangeRequest from 'dataparcels/ChangeRequest';
+import type {ParcelValueUpdater} from 'dataparcels';
+
+// $FlowFixMe - useState is a named export of react
```

Comment:
You can update flow to fix this error if you like

Reply:
Cool, I'll remember that. I'll upgrade once Allan upgrades blueflag-test